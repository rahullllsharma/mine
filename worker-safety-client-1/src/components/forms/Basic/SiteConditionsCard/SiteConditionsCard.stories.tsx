import type { ComponentMeta, ComponentStory } from "@storybook/react";

import { SiteConditionsCard } from "./SiteConditionsCard";

export default {
  title: "JSB/SiteConditionsCard",
  component: SiteConditionsCard,
} as ComponentMeta<typeof SiteConditionsCard>;

export const Playground: ComponentStory<typeof SiteConditionsCard> = args => (
  <SiteConditionsCard {...args} />
);

Playground.args = { title: "Traffic density" };
