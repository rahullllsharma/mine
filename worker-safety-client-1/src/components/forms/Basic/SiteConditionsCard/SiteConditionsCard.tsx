import classnames from "classnames";

import { TagCard } from "../TagCard";

export type SiteConditionsCardProps = {
  className?: string;
  title: string;
};

function SiteConditionsCard({ className, title }: SiteConditionsCardProps) {
  return (
    <TagCard className={classnames("border-data-blue-30", className)}>
      <span className="font-semibold">{title}</span>
    </TagCard>
  );
}

export { SiteConditionsCard };
