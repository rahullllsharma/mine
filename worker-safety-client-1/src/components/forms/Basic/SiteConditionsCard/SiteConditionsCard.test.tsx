import { render } from "@testing-library/react";

import { SiteConditionsCard } from "./SiteConditionsCard";

describe(SiteConditionsCard.name, () => {
  it("renders correctly", () => {
    const { asFragment } = render(
      <SiteConditionsCard title="traffic density" />
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
