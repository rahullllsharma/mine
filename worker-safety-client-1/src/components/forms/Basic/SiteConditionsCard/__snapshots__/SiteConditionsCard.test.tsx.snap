// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SiteConditionsCard renders correctly 1`] = `
<DocumentFragment>
  <div
    class="bg-white flex items-center shadow rounded-lg px-4 py-6 border-l-10 border-data-blue-30"
  >
    <span
      class="font-semibold"
    >
      traffic density
    </span>
  </div>
</DocumentFragment>
`;
